src/components/ContentPreview/ContentPreview.js

```diff
@@ -228,6 +235,13 @@ class ContentPreview extends PureComponent<Props, State> {
             currentFileId: fileId,
             prevFileIdProp: fileId,
         };
+        this.props.onMetric(
+            METRIC_TYPE_ELEMENTS_LOAD_METRIC,
+            {
+                endMarkName: MARK_NAME_JS_READY,
+            },
+            EVENT_JS_READY,
```

Comment:
It looks like `EVENT_JS_READY` is included in two places: as a suffix in `endMarkName` and here at the top level. Is there a way to avoid the duplication?

Reply:
Hmm.. the `Logger` can potentially inject multiple `on*` props. Maybe `onLoad` which can take `endMarkName` as a param, and automatically include `EVENT_JS_READY`. Thoughts?

Reply:
Yeah, I like that idea. It seems reasonable to use helper methods for commonly-logged metrics.